### PR TITLE
Fixes #6908: Finish 3.1 doc update

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/6908

### CODE REVIEW

File: 12_upgrade/00_upgrade.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 == Upgrade Rudder
 
 This short chapter covers the upgrade of the Rudder Server Root and Rudder Agent
-from older versions to the latest version, 3.0.
+from older versions to the latest version, 3.1.
 
 Please note that you can upgrade directly from Rudder 2.10.x or 2.11.x
```
Comment (VinceMacBuche) [inline 12_upgrade/00_upgrade.txt]: We can upgrade from 2.11 only in 3.1 (at least that what we want to have!

File: 12_upgrade/00_upgrade.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-to Rudder 3.0. However, direct upgrades from 2.9.x and older are no longer
+to Rudder 3.1. However, direct upgrades from 2.9.x and older are no longer
```
Comment (VinceMacBuche) [inline 12_upgrade/00_upgrade.txt]: from 2.10.X

File: 12_upgrade/05_caution.txt [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-Rudder 2.11.* use a compatible CFEngine version. Therefore, Rudder agent 2.11.* are fully compatible with Rudder server 3.0.*, so it is not necessary to update your agents to 3.0.*.
+Rudder 2.11.* and 3.0.* use a compatible CFEngine version. Therefore, Rudder agent 2.11.* and 3.0.* are fully compatible with Rudder server 3.1.*, so it is not necessary to update your agents to 3.1.*.
 
-To have a successful upgrade, you only need to upgrade Rudder server to 3.0.
- 
-==== Upgrading from Rudder 2.6 or 2.10
+However, Rudder 3.1 provides a new inventory signing feature which will only be enabled on nodes running Rudder 3.1 agents. On nodes running older agent versions, the inventories won't be signed. 
 
-Rudder 2.6.* and 2.10.* contain an older CFEngine version. To have a successful upgrade these steps should be followed:
+==== Upgrading from Rudder 2.10
```
Comment (VinceMacBuche) [inline 12_upgrade/05_caution.txt]: You can remove that section in fact :)

File: 11_install_agent/30_install_agent_suse.txt [hunk 4]
```diff
@@ -16,7 +16,7 @@ zypper ar -n 'Rudder SLES repository' http://www.rudder-project.org/rpm-3.0/SLES
 
 ----
 
-zypper sa 'http://www.rudder-project.org/rpm-3.0/SLES_10_SP3/' Rudder
+zypper sa 'http://www.rudder-project.org/rpm-3.1/SLES_10_SP3/' Rudder
```
Comment (VinceMacBuche) [inline 11_install_agent/30_install_agent_suse.txt]: Can you add a section about SLES12 here please ? 
zypper ar -n 'Rudder SLES 12 repository' http://www.rudder-project.org/rpm-3.1/SLES_12/ Rudder

**Comment by VinceMacBuche** [comment]
Really thanks you for that PR @amousset  ;)

**Comment by amousset** [comment]
PR updated.

File: 12_upgrade/05_caution.txt [hunk 5]
```diff
@@ -1,24 +1,10 @@
 === Caution cases
 
-==== Upgrading from Rudder 2.11
+==== Compatibility between Rudder server 3.1 and older agent versions
 
-Rudder 2.11.* use a compatible CFEngine version. Therefore, Rudder agent 2.11.* are fully compatible with Rudder server 3.0.*, so it is not necessary to update your agents to 3.0.*.
+Rudder 2.11.* and 3.0.* use a compatible CFEngine version. Therefore, Rudder agent 2.11.* and 3.0.* are fully compatible with Rudder server 3.1.*, so it is not necessary to update your agents to 3.1.*.
```
Comment (VinceMacBuche) [inline 12_upgrade/05_caution.txt]: Can you add a line to explain older version case: 
"To upgrade from oldest version, you have to upgrade to 3.0 or 2.11 before upgrading to 3.1"

**Comment by amousset** [comment]
PR updated.

**Comment by VinceMacBuche** [comment]
Great section about old release upgrade! thanks @amousset , merging!

**Pull request merged**